README.md

```diff
@@ -118,14 +118,28 @@ Compiling
 
 ### Compiling on GNU/Linux
 
-Install dependencies. Here's an example for Debian/Ubuntu:
+#### Dependencies
+
+| Dependency | Version | Commentary |
+|------------|---------|------------|
+| GCC        | 4.9+    | Can be replaced with Clang 3.4+ |
+| CMake      | 2.6+    |            |
+| Irrlicht   | 1.8.0+  |            |
+| SQLite3    | 3.8+    |            |
```

Comment:
pretty sure **any** version of sqlite3 is supported

Reply:
Really? Ok.